Unify ansible-galaxy install -r (#67843)
ansible/ansible: docs/docsite/rst/shared_snippets/installing_multiple_collections.txt

        version: 0.9.3
        source: https://galaxy.ansible.com

> To install both the roles and collections at the same time with 1 command, run the following

Reviewer: ```suggestion
To install both the roles and collections at the same time with single command, run the following
```

Author: I probably want to check with someone from the docs team as to whether we want to display the number or the number as a word.

Reviewer: sure. cc @samccann

Reviewer 2: Let's use "one" — see https://docs.ansible.com/ansible/latest/dev_guide/style_guide/grammar_punctuation.html#numbers

> Numbers between one and nine are written out. 10 and above are numerals.

.. code-block:: bash

    $ ansible-galaxy install -r requirements.yml

Running ``ansible-galaxy collection install -r`` or ``ansible-galaxy role install -r`` will only install collections

> and roles respectively.

Reviewer 2: ```suggestion
and roles, respectively.
```

.. note::

>     Installing both roles and collections from the same requirements file will not work when specifying a custom path

Reviewer: ```suggestion
    Installing both the roles and collections from the same requirements file will not work when specifying a custom path
```

Reviewer 2: It reads better to me without the extra `the` here.

>     to install either collections or roles to. In this scenario the collections will be skipped and the command will

Reviewer 2: ```suggestion
    collections or roles install path. In this scenario the collections will be skipped and the command will
```

(and removing 'path' from the line above).

>     process each like like ``ansible-galaxy role install`` would.

Reviewer 3: ```suggestion
    process each like ``ansible-galaxy role install`` would.
```

Little nit about the sentence before - does 'Installing both roles and collections from the same requirements file will not work when specifying a custom installation path.' read better?
